index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    |<header id="header">
+        <nav id="nav-bar">
+          <img id="header-img" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" alt="logo" class="logo">
+          <ul>
+            <li class="nav-link" >Features</li>
+            <li class="nav-link" >How It Works</li>
+            <li class="nav-link" >Pricing</li>
```

Comment:
Please add a href link to all the items you have placed in the navbar. Adding links to the navbar items is essential for them to navigate onto the specific topics they refer to.